### PR TITLE
Move `botGradleGitHubToken` into environment variable

### PR DESCRIPTION
**Pull request by blindpirate**
For security.

### CODE REVIEW

File: .teamcity/src/main/kotlin/promotion/PublishRelease.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ abstract class PublishRelease(
     triggerName = "ReadyforRelease",
     gitUserEmail = "%gitUserEmail%",
     gitUserName = "%gitUserName%",
-    extraParameters = "-PconfirmationCode=%confirmationCode%"
+    extraParameters =""" "-PconfirmationCode=%confirmationCode%" "-PbotGradleGitHubToken=%github.bot-gradle.token%" """
```
Comment (big-guy) [inline .teamcity/src/main/kotlin/promotion/PublishRelease.kt:33]: Does this cause it to be passed via the command line?

Comment (blindpirate) [reply .teamcity/src/main/kotlin/promotion/PublishRelease.kt]: You are right.

File: .teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt [hunk 2]
```diff
@@ -45,10 +45,14 @@ abstract class PublishGradleDistribution(
             gradleWrapper {
                 name = "Promote"
                 tasks = task
-                gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" "-PbotGradleGitHubToken=%github.bot-gradle.token%" """
+                gradleParams = """-PcommitId=%dep.${RelativeId("Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger")}.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail" """
```
Comment (wolfs) [inline .teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt:48]: This PR is confusing. It currently moves the command line argument to an environment variable, which I think is a good thing. The description says the PR does something else.

Moreover, I don't understand is why the PR is not against `release6x`.

**Comment by blindpirate** [comment]
@bot-gradle test and merge

File: .teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt (hunk 2 above)
Comment (blindpirate) [reply .teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt:48]: I'll merge this and cherry pick to `release`/`release6x`.

**Review by wolfs — APPROVED** [review]
LGTM!

**Pull request merged**